src/bn256/mod.rs

```diff
@@ -30,7 +30,7 @@ pub struct Bn256CeremonyParameters {
 }
 
 impl PowersOfTauParameters for Bn256CeremonyParameters {
-    const REQUIRED_POWER: usize = 10; // generate to have roughly 64 million constraints
+    const REQUIRED_POWER: usize = 26; // generate to have roughly 64 million constraints
```

Comment:
need to update comment, if this is supposed to stay

Reply:
It was for testing.
Now I am using:
`sed -i 's/const REQUIRED_POWER: usize = [0-9][0-9];*/const REQUIRED_POWER: usize = 8;/g' /app/src/bn256/mod.rs
`